### PR TITLE
PHPCBF on CSS/JS

### PR DESCRIPTION
**Pull request by paulschreiber**
* ensures newlines are \n
* space out JS functions
* add braces to JS functions

### CODE REVIEW

**Comment by paulschreiber** [comment]
@joshbetz?

File: js/co-authors-plus.js [hunk 1]
```diff
@@ -1,533 +1,504 @@
...
-if( typeof(console) === 'undefined' ) {
-	var console = {}
-	console.log = console.error = function() {};
-}
\ No newline at end of file
+jQuery( document ).ready(function () {
+
+	/*
+     * Click handler for the delete button
+     * @param event
```
Comment (mjangda) [inline js/co-authors-plus.js]: There's some weirdness happening with the indentation for docblocks in this file.

**Comment by paulschreiber** [comment]
@mjangda docblocks should be better now.

File: js/co-authors-plus.js [hunk 2]
```diff
@@ -1,533 +1,504 @@
...
+			*/
+			.suggest(coAuthorsPlus_ajax_suggest_link, {
+				onSelect: coauthors_autosuggest_select
+			})
+			.keydown( coauthors_autosuggest_keydown );
+
+			if (authorName) {
+				$co.attr( 'value', unescape( authorName ) ); } else {
+				$co.attr( 'value', coAuthorsPlusStrings.search_box_text )
+				.focus( function(){ $co.val( '' ) } } )
```
Comment (mjangda) [inline js/co-authors-plus.js]: There's some weirdness with the automated cleanup. For example, There's now an extra `}` here, which will probably break things.

Before:

```
.focus( function(){ $co.val( '' ) } )
```

After:

```
.focus( function(){ $co.val( '' ) } } )
```

There may be other similar issues. Might be better to approach this cleanup in smaller chunks?

Comment (paulschreiber) [inline js/co-authors-plus.js]: Looks like PHPCS is unreliable for JS. I'll delete this branch and redo with JSCS.

**Comment by paulschreiber** [comment]
Closing this. Will redo.

**Pull request closed without merging**